lib/core/test/unit/Cardano/Wallet/Primitive/CoinSelection/MA/RoundRobinSpec.hs

```diff
@@ -289,8 +290,8 @@ prop_runSelection_UTxO_moreThanEnough (Small index) = monadicIO $ do
         cutAssetSetSizeInHalf balanceAvailable
 
 prop_runSelection_UTxO_muchMoreThanEnough
-    :: Large UTxOIndex -> Property
-prop_runSelection_UTxO_muchMoreThanEnough (Large index) =
+    :: Blind (Large UTxOIndex) -> Property
```

Comment:
Good idea. 👍🏻  I was not aware of the `Blind` wrapper type.

In the same spirit, even the "small" UTxO indices can be hard to read (when output through `show`). Perhaps it's worth putting `Blind` on all of them.

Reply:
Generally speaking, the default `Show` instance is good for small data-types. When data-structures become complex, it is just unreadable (though sometimes useful to have a copy-pastable counter-example). As a rule of thumb, I prefer using few counter-examples with cherry-picked pieces of information that makes asserting the problem easier.